### ISSUE TITLE
`dict[str, list[OmeroChannel]]` data not displayed correctly

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#208]
1. Collect fractal-tasks-core 0.10.0a6
2. Create a workflow, and add task `Create OME-ZARR structure (multiplexing)`
3. Fill the `allowed_channels` required arguments until "save" works. This will require adding a key-value pair, and in the value there should be an item, and in the item one should set `wavelength_id="aaa"` and one should set `window["start"]=0` and `window["end"]=1`
4. Save -> fractal-web shows no error
5. EXPECTED BEHAVIOR: I should see the new element in allowed channels. ACTUAL BEHAVIOR: there is nothing displayed.

Note that the DB has the right data, as is clear e.g. by exporting the worfklow:
```json
      "args": {
        "image_extension": "tif",
        "num_levels": 5,
        "coarsening_xy": 2,
        "allowed_channels": {
          "asd": [
            {
              "wavelength_id": "asd",
              "window": {
                "start": 0,
                "end": 1
              },
              "active": true,
              "coefficient": 1,
              "inverted": false
            }
          ],
          "new": [
            {
              "wavelength_id": "asd",
              "window": {
                "start": 1,
                "end": 1
              },
              "active": true,
              "coefficient": 1,
              "inverted": false
            }
          ]
        }
```

**Comment by tcompa** [comment]
![Uploading Screenshot from 2023-06-22 16-47-09.png…]()

**Comment by tcompa** [comment]
Fixed with 57d0664, and I confirm that the behavior is as expected.
